sensor_nodes/wot-config/bindings.json

```diff
@@ -81,33 +81,6 @@
                 }
             ]
         },
-        "temperature_range": {
```

Comment:
I'm not sure, but I think this results in an error because the form for  `temperature_range` is missing.

Reply:
Hmm, I think this should not be a problem as affordances that do not have a form in the consolidated version *should* be ignored.